Client/WarFare/UISkillTreeDlg.h

```diff
@@ -18,10 +18,17 @@ const int SKILL_DEF_SPECIAL1 = 2;
 const int SKILL_DEF_SPECIAL2 = 3;
 const int SKILL_DEF_SPECIAL3 = 4;
 
-// MAX_SKILL_KIND_OF	:						 Base Skill 1개, 전문 스킬 4개..			
-// MAX_SKILL_IN_PAGE	:						 한 페이지 내의 아이콘 갯수..				
-// MAX_SKILL_PAGE_NUM	:						 스킬 한개가 가지는 최대 페이지수..		
-
+// Master Scrolls (items consumed)
+constexpr DWORD ITEM_ID_MASTER_SCROLL_WARRIOR		= 379063000;
+constexpr DWORD ITEM_ID_MASTER_SCROLL_ROGUE		= 379064000;
+constexpr DWORD ITEM_ID_MASTER_SCROLL_MAGE		= 379065000;
+constexpr DWORD ITEM_ID_MASTER_SCROLL_PRIEST		= 379066000;
+
+// Skill IDs requiring dual/double weapons
+constexpr int SKILL_ID_DUAL_WEAPON_WARRIOR		= 1055;
```

Comment:
These aren't exactly 'skill IDs' as we know them. This is a confusing name in this respect.
A 'skill ID' is a 6 digit ID referencing a skill in the various magic tables.

I understand it's used by `iNeedSkill` but a distinction needs to be made here for clarity.